packages/stark-ui/src/util/component/component.util.spec.ts

```diff
@@ -26,4 +28,70 @@ describe("Util: ComponentUtil", () => {
 			expect(result).toBe(false);
 		});
 	});
+
+	describe("isOutputWiredUp", () => {
+		let component: TestComponent;
+
+		beforeEach(async(() => {
+			return TestBed.configureTestingModule({
+				declarations: [TestHostWiredUpComponent, TestHostNotWiredUpComponent, TestComponent]
+			}).compileComponents();
+		}));
+
+		it("shouldReturnTrue", () => {
```

Comment:
Maybe a more descriptive test will be useful... when it is supposed to return true? ;)